config/prow/config.yaml

```diff
@@ -515,6 +515,9 @@ tide:
     kubernetes-sigs/minibroker: squash
     kubernetes/cloud-provider-openstack: squash
     kubernetes/dashboard: squash
+  merge_commit_template:
```

Comment:
Should we do this consistently for all repos?

Reply:
https://github.com/kubernetes/test-infra/issues/17915 has all the context why kubeflow/pipelines want this.
but I think it also has the side effect of dropping commit body all together (github default behavior is concatenating all commits' messages and bodies). It's fine for KFP, we don't care about commit body anyway.

If you think that seems reasonable for other repos too, I have no objection, but maybe it's better we first try this configuration in KFP to see if it's working as expected, (because looks like although the feature was implemented, no one was using it before).